desktop/js_files/package.json

```diff
@@ -60,13 +61,14 @@
     "qrcode": "^1.4.1",
     "react": "16.6.1",
     "react-dom": "16.4.2",
-    "react-native": "git+https://github.com/status-im/react-native-desktop.git#v0.57.8_10",
+    "react-native": "git+https://github.com/status-im/react-native-desktop.git#master",
```

Comment:
Before being merged, this reference should be changed from `master` to a tag or a commit hash.